docs/0.1/index.md

```diff
@@ -99,13 +99,15 @@ To address this use case, the provider creates and makes available a list of URI
 
 In some cases, additional metadata is also cached as part of the external reference, but this is less common. Verification of the additional metadata may require the consumer to take additional actions.
 
-#### 1.2.3. Local Cache of Full Dataset
+#### 1.2.3. Local Cache of Dataset of Full Entity Metadata
```

Comment:
Could we change the title to simply "Local Cache of Full Entity Metadata", at present it reads very awkwardly

Reply:
Updated the pull request to incorporate this suggestion